sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSqlDialectSpecTest.java

```diff
@@ -346,7 +346,7 @@ public void testIn() {
 
     PCollection<Row> stream = execute(sql);
 
-    final Schema schema = Schema.builder().addNullableField("field1", FieldType.BOOLEAN).build();
+    Schema.builder().addNullableField("field1", FieldType.BOOLEAN).build();
```

Comment:
nit: This doesn't have side effects, you can just delete it.

Reply:
Done